packages/components/src/ColorPicker/useColorPicker.js

```diff
@@ -1,37 +1,157 @@
 import { useContextSystem } from '@wp-g2/context';
-import { cx } from '@wp-g2/styles';
-import { noop } from '@wp-g2/utils';
+import { ui } from '@wp-g2/styles';
+import { noop } from 'lodash';
+import { useCallback, useState } from 'react';
 
-import * as styles from './ColorPicker.styles';
-import { useColorPickerState } from './useColorPickerState';
+export function getColor(color) {
+	return ui.color(color).toRgbString();
+}
+
+/**
+ * @param {State} state
+ * @return {string}
+ */
+const getColorValue = (color, inputType) => {
+	let colorValue = color;
+
+	switch (inputType) {
+		case 'hex':
+			colorValue = ui.color(color).toHexString();
+			break;
+		case 'rgb':
+			colorValue = ui.color(color).toRgbString();
+			break;
+		case 'hsl':
+			colorValue = ui.color(color).toHslString();
+			break;
+		default:
+			break;
+	}
+
+	return colorValue;
+};
 
 export function useColorPicker(props) {
 	const {
-		color,
+		// @todo: Add support to interpret initial alpha + color.
```

Comment:
Does this todo still need to happen or is it out of scope for this PR?

Reply:
I think it's out of scope for this PR 👍